chapters/operatorsandexpressions.tex

```diff
@@ -1158,10 +1158,12 @@ \subsection{Event-Related Operators with Function Syntax}\doublelabel{event-rela
 positive number.\\ \hline
 
 % pre
-\lstinline!pre(y)! & Returns the \emph{left limit} $y(t\textsuperscript{pre})$ of
+\lstinline!pre(y)! & Returns the \emph{left limit} $y(t^{-})$ of
 variable $y(t)$ at a time instant $t$. At an event instant,
-$y(t\textsuperscript{pre})$ is the value of y after the last event
-iteration at time instant $t$ (see comment below). The \lstinline!pre()! operator can
+$y(t^{-})$ is the value of y after the last event
```

Comment:
Based on the other occurrences of "y", shouldn't this one also be `$y$`?

Reply:
Yes, will change